drivers/wifi/simplelink/simplelink.c

```diff
@@ -24,6 +24,9 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define SCAN_RETRY_DELAY 2000  /* ms */
 #define FC_TIMEOUT K_SECONDS(CONFIG_WIFI_SIMPLELINK_FAST_CONNECT_TIMEOUT)
 
+#define IPV4 0x1
```

Comment:
I think there is a chance that defines with the same name are used somewhere else. I would prefer if these defines here would be prefixed, e.g. `IPV4` -> `SIMPLELINK_IPV4`